Add 2.7 release notes (#7661)
grafana/loki: docs/sources/release-notes/v2-7.md

---
title: V2.7
weight: 66
---

# Version 2.7 release notes

> Grafana Labs is excited to announce the release of Loki 2.7. Here's a summary of new enhancements and important fixes.

Reviewer: ```suggestion
Grafana Labs is excited to announce the release of Loki 2.7. Here's a summary of new enhancements and important fixes:
```

## Features and enhancements

> - **New Internal Server and TLS** TLS can now be configured everywhere, and there is an internal server to allow ring, memberlist, and readiness handlers to continue to work without TLS.

Reviewer: ```suggestion
- **New Internal Server and TLS** TLS can now be configured everywhere and there is an internal server to allow ring, memberlist, and readiness handlers to continue to work without TLS.
```

> - **Better Support for Azure Blob Storage** thanks to the ability to use Azuure's Service Prinicipal credentials.

Reviewer: ```suggestion
- **Better Support for Azure Blob Storage** thanks to the ability to use Azure's Service Principal Credentials.
```
